jetty9-base/src/main/java/com/google/cloud/runtimes/jetty9/TracingLogHandler.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.runtimes.jetty9;
+
+import java.util.logging.ConsoleHandler;
+import java.util.logging.LogRecord;
+
+/**
+ * A Logging Handler extended with a request traceId label.
+ */
+public class TracingLogHandler extends ConsoleHandler {
+
+  /**
+   * Construct a TracingLogHandler.
+   */
+  public TracingLogHandler() {}
+
+  @Override
+  public synchronized void publish(LogRecord record) {
+    String traceId = RequestContextScope.getCurrentTraceId();
+    if (traceId != null) {
+      record.setMessage(record.getMessage() + " traceId=" + traceId);
```

Comment:
@meltsufin's comment was hidden when I rename to `traceId`.  Two questions remain:
 1. should we add traceId to the log record parameters and use a special formatter?
 2. should the non jetty specific JUL classes such as formatter be in `openjdk-runtime`?